packages/mdc-switch/_mixins.scss

```diff
@@ -192,6 +192,24 @@
   }
 }
 
+// Opacity customizations apply to both on and off states to ensure symmetry.
```

Comment:
Follow SassDoc convention. Also add docs for param using `@param` annotation.

See http://sassdoc.com/annotations/